flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -289,3 +297,10 @@ class SentryFlutterOptions extends SentryOptions {
   /// The [navigatorKey] is used to add information of the currently used locale to the contexts.
   GlobalKey<NavigatorState>? navigatorKey;
 }
+
+/// Callback being executed in [ScreenshotEventProcessor], deciding if a
+/// screenshot should be recorded and attached.
+typedef BeforeScreenshotCallback = FutureOr<bool> Function(
+  SentryEvent event, {
+  Hint? hint,
```

Comment:
@denrase question: should hint be nullable here? afaik we removed nullable hints

Reply:
if we don't need it we can update it in the v8 branch

Reply:
Lets check this in a separate issue.